.github/workflows/ci.yml

```
@@ -122,14 +122,14 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.8", "3.11"]
+        python-version: ["3.8", "3.12"]
         db-backend: ["postgresql"]
         nautobot-version: ["stable"]
         include:
-          - python-version: "3.11"
+          - python-version: "3.12"
```

This one needs to stay on 3.11 since nautobot v2.0.0 doesn't support 3.12

```suggestion
          - python-version: "3.11"
```